src/utils/TodoItem.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Todo } from '../types/Todo';
+
+interface TodoItemProps {
+  todo: Todo;
+  onDoubleClick?: () => void;
+  onClick?: (id: number) => void;
+  isLoading: boolean,
+}
+
+export const TodoItem: FC<TodoItemProps> = (
+  {
+    todo, onDoubleClick, onClick, isLoading,
+  },
+) => {
+  const onClickHandler = () => {
+    onClick?.(todo.id);
+  };
+
+  return (
+    <div
+      className={classNames('todo', {
+        completed: todo.completed,
+      })}
+      onDoubleClick={onDoubleClick}
+
+    >
+      <label className="todo__status-label">
+        <input type="checkbox" className="todo__status" checked />
```

Comment:
The checkbox input is always checked. It should be dependent on the 'completed' status of the todo. Use `checked={todo.completed}` instead of `checked`.